elasticsearch_dsl/faceted_search.py

```diff
@@ -130,6 +130,7 @@ class DateHistogramFacet(Facet):
     agg_type = 'date_histogram'
 
     DATE_INTERVALS = {
+        'year': lambda d: (d+timedelta(days=366)).replace(day=1),
```

Comment:
this should be simpler, just `d.replace(year=d.year+1)` - no need for the complex expression since years are strictly linear (unlike months).

Reply:
Should be done.